optimize_versioned_docs/version-3.7.0/components/userguide/process-analysis/report-analysis/edit-mode.md

```diff
@@ -46,19 +46,27 @@ In this section of the report builder, you are characterizing the output of the
 
 First, you need to select which part of the data you want to view. Optimize differentiates between the view (e.g. process instance or flow node) and the measure (e.g. count or duration):
 
-1. Raw Data: View just a table with the actual data listed as rows. This can come in handy if you found interesting insights in certain process instances and need detailed information about those instances, or you are exploring a process definition with a limited number of instances. This report type also allows you to inspect raw [object variable values](../../../../../self-managed/optimize-deployment/setup/object-variables.md).
+1. Raw Data: View just a table with the actual data listed as rows. This can come in handy if you found interesting insights in certain process instances and need detailed information about those instances, or you are exploring a process definition with a limited number of instances. This report type also allows you to inspect raw [object variable values](../../../../self-managed/optimize-deployment/setup/object-variables.md).
 2. Process instance
-  * Count: View how many process instances were executed.
-  * Duration: View how long the process instances took to complete.
+
+- Count: View how many process instances were executed.
+- Duration: View how long the process instances took to complete.
+
```

Comment:
I was worried about this difference in formatting, that it might change the indentation...but it looks like there's no visual change: 

<img width="1478" alt="image" src="https://user-images.githubusercontent.com/1627089/196528828-37285f4c-31d7-4520-87fb-d657e2fb1d51.png">

Reply:
Thanks for the screenshot!